packages/@vue/cli-service/lib/config/app.js

```diff
@@ -140,7 +140,7 @@ module.exports = (api, options) => {
     const multiPageConfig = options.pages
     const htmlPath = api.resolve('public/index.html')
     const defaultHtmlPath = path.resolve(__dirname, 'index-default.html')
-    const publicCopyIgnore = ['index.html', '.DS_Store']
+    const publicCopyIgnore = ['/index.html', '.DS_Store']
```

Comment:
Actually we can just delete the `'/index.html'` pattern…
It doesn't match any files as I tested out. But it doesn't break anything because that html-webpack-plugin's output overwrites copy-webpack-plugin's anyway…

Reply:
Oh, thought copy would run after the html plugin